pkg/pod/status.go

```diff
@@ -247,7 +247,15 @@ func areStepsComplete(pod *corev1.Pod) bool {
 	return stepsComplete
 }
 
+func sortContainerStatuses(podInstance *corev1.Pod) {
```

Comment:
It would be nice to have unit tests for this

Reply:
@afrittoli 
Done, thanks